deploy/kubernetes/helm/che/custom-charts/che-devfile-registry/templates/ingress.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: che-devfile-registry
+  annotations:
+    kubernetes.io/ingress.class: {{ .Values.cheDevfileRegistry.ingress.class }}
+spec:
+  rules:
+  - host: {{ printf .Values.global.cheDevfileRegistryUrlFormat .Release.Namespace .Values.global.ingressDomain }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: che-devfile-registry
+          servicePort: 8080
+{{- if .Values.cheDevfileRegistry.ingressSecretName }}
+  tls:
+  - hosts:
+    - {{ printf .Values.global.cheDevfileRegistryUrlFormat .Release.Namespace .Values.global.ingressDomain }}
+    secretName: {{ .Values.cheDevfileRegistry.ingressSecretName }}
```

Comment:
In the operator, the same `secretName` setting (even most of the same ingress settings) is used for all ingresses: the Che one, the Keycloak one, and now the registry ones.
Does it have sense to have a distinct secret name for each registry ingress ?

Reply:
fixed. now it uses `global.tls.secretName`

Reply:
Thanks